Status/Day_21.md

```diff
@@ -183,6 +183,10 @@ ans = list(removeDuplicate(li))
 print(ans)
 ```
 
+```python Solution by - Utkarsh4697
+li = [12,24,35,24,88,120,155,88,120,155]
+print(sorted(list(set(li))))

```

Comment:
Thanks for your contribution. But it seems your code doesn't give the output as mentioned **original order reserved** rather it's giving sorted order. Please fix it and pull the changes again. :)